net/ntopng/Makefile

```diff
@@ -0,0 +1,8 @@
+PLUGIN_NAME=		ntopng
+PLUGIN_VERSION=		0.1
+PLUGIN_COMMENT=		Traffic Analysis and Flow Collection
+PLUGIN_DEPENDS=		ntopng
```

Comment:
If redis is required, make it a hard dependency (it looks like you did not have to configure it here but receive a warning when its is missing in the gui)

Reply:
I can add it as a dependency but it won't run without enabling it via rc.conf, so why put as a dependency?

Reply:
You can depend on os-redis to require the plugin. If you want you can use its model to enable it when you enable your plugin

Reply:
We had this discussion with C-ICAP and ClamAV already, @fichtner said it's problematic with cross plugin dependencys. It the same thing as with C-ICAP.

Reply:
@fichtner it is a bad idea because it impies a conflict between os-ntopng and os-redis since both plugins would use the same resource (redis config file)

Reply:
redis is hard dependency, also ntopng wont start without a running redis. Thats why I added the warning. I dont see a problem when multiple instances use redis, that's why we have it as a separate plugin :)